Mergin/plugin.py

```diff
@@ -623,6 +703,21 @@ def actions(self, parent):
         return actions
 
 
+class MerginProjectsManager(object):
```

Comment:
Could you add some docstrings with brief class description?

Reply:
Done